### ISSUE TITLE
ObjectInvocation found on PHPUnit 8.2.1

### ISSUE THREAD

**Issue by llstarscreamll** [blastcloud/guzzler#9]
Hi, thank you so much for this awesome package!!

Running on PHPUnit 8.2.1 the following error is thrown:

```
[Error] Class 'PHPUnit\Framework\MockObject\Invocation\ObjectInvocation' not found
#1  /home/johan/Code/pascal/vendor/blastcloud/guzzler/src/Expectation.php:141
#2  /home/johan/Code/pascal/vendor/blastcloud/guzzler/src/Guzzler.php:43
...
```

The `PHPUnit\Framework\MockObject\Invocation` namespace was removed on version [v8.2](https://github.com/sebastianbergmann/phpunit/tree/8.2.0/src/Framework/MockObject) but remains on [v8.1.6](https://github.com/sebastianbergmann/phpunit/tree/8.1.6/src/Framework/MockObject).

**Comment by adamkelso** [comment]
1. Thanks! I'm very glad it's been helpful.
2. Huh. That's interesting. Thanks for bringing it up. Upgrading on my end to the latest to check it out and find a solution.

**Comment by adamkelso** [comment]
@llstarscreamll Thanks for reporting the bug. Just released version `1.5.2`, which fixes the issue.

Also, would love any feedback on the proposed issues, if you've got the time. Thanks again.

**Comment by llstarscreamll** [comment]
Hi @adamkelso, everything is ok now. Thank you so much!!
